docs/maps/import-geospatial-data.asciidoc

```diff
@@ -0,0 +1,32 @@
+[role="xpack"]
+[[import-geospatial-data]]
+== Import geospatial data
+
+Geospatial data in the Elastic Stack must be indexed as a {ref}/geo-point.html[geo_point] or {ref}/geo-shape.html[geo_shape].
+Kibana makes it easy to import geospatial data into the Elastic Stack.
```

Comment:
How about using the intro to explain that Kibana offers two ways to import geospatial data and the benefit of each.  Roughly 3 sentences with the following:

1. Kibana offers two way to import geospatial data into the Elastic Stack.
2. Explain CSV upload and why you would want that.
3. 2. Explain GeoJSON upload and why you would want that.